test/test_about_me.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+require_once('../admin_functions.php');
+
+use PHPUnit\Framework\TestCase;
+
+class admin extends TestCase {
+
+    public function testfetchContentSuccess() {
+        $expected = "My name is Dan and I am a Trainee Full Stack Developer";
+        $testarray = ["text"=>"My name is Dan and I am a Trainee Full Stack Developer"];
+        $key = "text";
+        $case = fetchContent($key, $testarray);
+        $this->assertEquals($case, $expected);
+    }
+
+    public function testfetchContentFailures() {
+        $expected = "Error incorrect input";
+        $testarray = ["text"=>"My name is Dan and I am a Trainee Full Stack Developer"];
+        $case = fetchContent('poop', $testarray);
+        $this->assertEquals($case, $expected);
```

Comment:
no test for what happens if you pass an array as the first param to fetchContent()